src/main/scala/org/ergoplatform/mining/ErgoMiner.scala

```diff
@@ -239,9 +253,9 @@ class ErgoMiner(ergoSettings: ErgoSettings,
   private def generateCandidate(h: ErgoHistoryReader,
                                 m: ErgoMemPoolReader,
                                 s: UtxoStateReader,
-                                txsToIncludeOpt: Option[Seq[ErgoTransaction]]): Try[CandidateCache] = {
+                                txsToInclude: Seq[ErgoTransaction]): Try[CandidateCache] = {
     //mandatory transactions to include into next block taken from the previous candidate
-    val txsToInclude = txsToIncludeOpt.getOrElse(candidateOpt.map(_.txsToInclude).getOrElse(Seq.empty)).filter { tx =>
+    val unspentTxsToInclude = txsToInclude.filter { tx =>
```

Comment:
`Option` was legacy here, it was always `Some`

Reply:
Good catch!